File-Formats/Presentation/create-read-edit-master-slide-and-layout-slide-in-c-sharp.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Create, read and edit Master and Layout slides in CSharp |Syncfusion|
-description: Create, read and edit Master slides and Layout slides in CSharp
+description: Create, read and edit Master slides and Layout slides in CSharp using Syncfusion PowerPoint Presentation library.
```

Comment:
Create, read and edit Master slides and Layout slides using Syncfusion PowerPoint library (Essential Presentation).

Reply:
Addressed